mmv1/third_party/terraform/services/container/resource_container_cluster_test.go.erb

```diff
@@ -9443,3 +9533,189 @@ func testAccContainerCluster_withWorkloadALTSConfig(projectID, name, networkName
 `, projectID, networkName, subnetworkName, name, enable)
 }
 <% end -%>
+
+func testAccContainerCluster_resourceManagerTags(projectID, clusterName, networkName, subnetworkName, randomSuffix string) string {
+	return fmt.Sprintf(`
+data "google_project" "project" {
+  project_id = "%[1]s"
+}
+
+resource "google_project_iam_binding" "tagHoldAdmin" {
+  project = "%[1]s"
+  role    = "roles/resourcemanager.tagHoldAdmin"
+  members = [
+    "serviceAccount:service-${data.google_project.project.number}@container-engine-robot.iam.gserviceaccount.com",
+  ]
+}
+
+resource "google_project_iam_binding" "tagUser" {
+  project = "%[1]s"
+  role    = "roles/resourcemanager.tagUser"
+  members = [
+    "serviceAccount:service-${data.google_project.project.number}@container-engine-robot.iam.gserviceaccount.com",
+    "serviceAccount:${data.google_project.project.number}@cloudservices.gserviceaccount.com",
+  ]
+}
+
+resource "time_sleep" "wait_120_seconds" {
+  create_duration = "300s"
+
+  depends_on = [
+    google_project_iam_binding.tagHoldAdmin,
+    google_project_iam_binding.tagUser
+  ]
+}
+
+resource "google_tags_tag_key" "key" {
+  parent = "projects/%[1]s"
+  short_name = "foobarbaz-%[2]s"
+  description = "For foo/bar resources"
+  purpose = "GCE_FIREWALL"
+  purpose_data = {
+    network = "%[1]s/%[4]s"
+  }
+}
+
+resource "google_tags_tag_value" "value" {
+  parent = "tagKeys/${google_tags_tag_key.key.name}"
+  short_name = "foo-%[2]s"
+  description = "For foo resources"
+}
+
+data "google_container_engine_versions" "uscentral1a" {
+  location = "us-central1-a"
+}
+
+resource "google_container_cluster" "primary" {
+  name               = "%[3]s"
+  location           = "us-central1-a"
+  min_master_version = data.google_container_engine_versions.uscentral1a.release_channel_latest_version["STABLE"]
+  initial_node_count = 1
+
+  node_config {
+    machine_type    = "n1-standard-1"  // can't be e2 because of local-ssd
+    disk_size_gb    = 15
+
+    resource_manager_tags = {
+      "tagKeys/${google_tags_tag_key.key.name}" = "tagValues/${google_tags_tag_value.value.name}"
+    }
+  }
+
+  deletion_protection = false
+  network    = "%[4]s"
+  subnetwork    = "%[5]s"
+
+  timeouts {
+    create = "30m"
+    update = "40m"
+  }
+
+  depends_on = [time_sleep.wait_120_seconds]
+}
+`, projectID, randomSuffix, clusterName, networkName, subnetworkName)
+}
+
+// TODO uncomment for auto_pilot_config
```

Comment:
Can you remove this from this PR? It can be added in a later PR if needed

Reply:
removed